core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -304,6 +314,15 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     }
 
     handleDragEvents() {
+        this.store.currentState$.pipe(takeUntil(this.destroy$)).subscribe((state) => {
+            if (state === EDITOR_STATE.DRAGGING) {
```

Comment:
Don't we have a scrolling selector?

Reply:
No, Scroll is a EDITOR_STATE item
Maybe we can create a new selector with filter, but I don't think it's necessary now